q2_sample_classifier/tests/test_classifier.py

```diff
@@ -543,7 +543,8 @@ def test_pd_dataframe_to_Probabilities_format(self):
                            columns=['classA', 'classB'],
                            index=['a', 'b', 'c', 'd'])
         obs = transformer(exp)
-        obs = pd.DataFrame.from_csv(str(obs), sep='\t', header=0)
+        obs = pd.read_csv(str(obs), sep='\t', header=0, index_col=0,
+                          parse_dates=True)
```

Comment:
I'm not sure we want pandas to be parsing dates on our behalf, that sounds like a recipe for unpredictable behavior. @nbokulich ?

Reply:
true. This is just in the unit tests, though, and it appears to work. I assumed perhaps this was added to make tests pass (e.g., maybe `from_csv` was parsing dates by default?)

Reply:
@David-Rod - why did you add this parameter?

Reply:
This [link](https://pandas.pydata.org/pandas-docs/version/0.21/generated/pandas.DataFrame.from_csv.html#pandas.DataFrame.from_csv) talked about including parameter values that would be needed to return a series similar to what from_csv accomplished.

I mistook these as necessary keyword argument values, rather than suggested values in place of defaults or other values. I will remove them from the function call.

Reply:
Thanks for sharing that link, I think this part sums up the motivating factor nicely:

> So a pd.DataFrame.from_csv(path) can be replaced by pd.read_csv(path, index_col=0, parse_dates=True).

So my understanding is that you were recreating the old `from_csv` behavior, which sounds good to me! In the future I think we can drop this flag, but no harm in keeping it for now. Thanks for explaining!